src/tox/session/commands/help.py

```diff
@@ -10,5 +10,8 @@ def show_help(config):
         "TOX_TESTENV_PASSENV: space-separated list of extra environment variables to be "
         "passed into test command environments"
     )
-    reporter.line("PY_COLORS: 0 disable colorized output, 1 enable (default)")
     reporter.line("TOX_PARALLEL_NO_SPINNER: 1 disable spinner for CI, 0 enable (default)")
+    reporter.line(
+        "tox depends on py.io.TerminalWriter to handle colouring, which supports forcing "
```

Comment:
I don't think we want to disclose this implementation behaviour as a report, please remove this change 👍

Reply:
Done.